src/box.tsx

```diff
@@ -43,19 +43,25 @@ const Box = React.forwardRef<HTMLDivElement, Props>(
         },
         ref
     ) => {
+        const paddingStyles = {paddingTop, paddingBottom, paddingLeft, paddingRight};
+        let paddingClasses = '';
+        try {
+            paddingClasses = sprinkles(paddingStyles);
```

Comment:
do we really need this? I know that we have a 24px in webapp, but I think that value should be changed to 24 or 32, if not this change could also be applied to Stack and other components

Reply:
I prefer to be safe here for the moment